src/index.scss

```
@@ -25,3 +25,32 @@ html, body {
 .gap-5 { gap: $spacer * 2 !important; }
 
 .gap-6 { gap: $spacer * 3 !important; }
+
+/* datepicker */
+
+.p-datepicker {
+  .p-datepicker-footer {
+    margin-top: .5rem;
+
+    & .p-button {
```

```suggestion
    .p-button {
```
тут же не нужен `&` по идее, `.p-button` и так вложен

---

Ага. Все так. Забрал из 4к просто их)